components/MyLayout.js

```diff
@@ -1,6 +1,6 @@
 import Head from 'next/head';
 
-import NavBar from './Navbar';
+import NavBar from './NavBar';
```

Comment:
What eslint rules are you using in here?

Reply:
Just added mastarm linting which uses standard.